.github/workflows/checks.yml

```diff
@@ -1,5 +1,10 @@
 name: Checks
 
+env:
+  STARKNET_VERSION: "0.12.3"
+  RPC_SPEC_VERSION: "0.5.1"
+  DEVNET_SHA: "78527de"
+
```

Comment:
how do we set this locally, if someone wants to install devnet?

Reply:
At this moment all the installation is on the developer side, tests will use `starknet-devnet` that is added to `PATH` (or fail if it is not found)

Reply:
yup, but how should the developer know which hash/rpc/starknet version to use? is it docummented somewhere? (sorry if it is, I couldn't find it)